libs/langchain/langchain/vectorstores/dashvector.py

```diff
@@ -29,15 +29,15 @@ class DashVector(VectorStore):
     Example:
         .. code-block:: python
 
-            from langchain.vectorstores import dashvector
+            from langchain.vectorstores import DashVector
```

Comment:
the imported `dashvector` had conflic with the other `dashvector` import below, and was inconsistent with the `DashVector(...)` code below.